src/lib/check.js

```diff
@@ -106,3 +106,19 @@ export const isNullUndefinedOrEmptyString = toBeAny([undefined, null, '']);
 export const isFunction = fun => typeof fun === 'function';
 
 export const hasOwnProperty = (object, propertyName) => Object.prototype.hasOwnProperty.call(object, propertyName);
+
+// The logical mode to use when having multiple filters.
+// Default is AND.
+// See https://docs.dhis2.org/master/en/developer/html/webapi_metadata_object_filter.html
```

Comment:
These comments would be good to get on a file-level in jsdoc. Maybe use a module documentation string?

Reply:
Been added to the logicMode() function.